modules/tinymce/src/themes/silver/main/ts/ui/core/complex/BlocksBespoke.ts

```diff
@@ -44,7 +45,7 @@ const getSpec = (editor: Editor): SelectSpec => {
   const dataset = buildBasicSettingsDataset(editor, 'block_formats', Delimiter.SemiColon);
 
   return {
-    tooltip: Tooltip.getTooltipText(editor, title, fallbackFormat),
+    tooltip: Tooltip.makeTooltipText(editor, `${btnTitle} {0}`, fallbackFormat),
```

Comment:
That's not quite what I meant (I would prefer to have an explicit string that is literally `Block {0}` with no string append _at all_) but this is better than it was.

Reply:
I feel this string append is necessary to make the code more maintainable. I am trying to make sure that the label is of the format "<name of component> <current value>", and the name of the component is `btnTitle`. If we were to change the name in the future we would have to remember to also update the tooltip manually if we had an explicit string instead.

Reply:
I understand, there are pros and cons to every approach, for example this could run into the plural issue for the same reason the 6.8 release did. But at least each case is all in one file now.

Reply:
Yeah that's a fair point that the tooltip should make sense on its own regardless of any slight changes to the name. I'm happy to make the change by replacing the `btnTitle` usages with a `btnTooltip` variable that is the explicit string. 

I'll then rename the `title` parameter to `tooltip` in the common `createSelectButton` function, since it is only used for that purpose, to remove the string append there as well:
https://github.com/tinymce/tinymce/blob/29a335850b6a0531498a559a380f439e8a1b069e/modules/tinymce/src/themes/silver/main/ts/ui/core/complex/BespokeSelect.ts#L190-L191

Reply:
Changes applied